keras/src/layers/core/masking.py

```diff
@@ -45,6 +46,9 @@ class Masking(Layer):
 
     def __init__(self, mask_value=0.0, **kwargs):
         super().__init__(**kwargs)
+        # `mask_value` can be a serialized tensor, hence verify it
+        if isinstance(mask_value, dict) and mask_value.get("config", None):
```

Comment:
You can actually remove this line! `deserialize_keras_object` should work in all circumstances.